contents/docs/user-guides/feature-flags.md

```diff
@@ -114,3 +114,71 @@ For feature flags that filter by user properties only, a given flag will always
 However, for flags using a rollout percentage mechanism (either by itself or in combination with user properties), the flag will persist for a given user as long as the rollout percentage and the flag key are not changed. 
 
 As a result, keep in mind that changing those values will result in flags being toggled on and off for certain users in a non-predictable way. 
+
+## A/B testing with feature flags (beta)
+
+PostHog 1.28 introduces support for A/B feature flags which can return string values according to a specified distribution. This is ideal for when you want to test multiple variants of the same interchangeable content, such as marketing taglines, colors, or page layouts. Currently, this is a beta feature for paying customers. Contact us through one of our [support options](https://posthog.com/support) to try this out!
+
+### Creating an A/B feature flag with multiple variants
+
+Create an A/B feature flag just like you would a standard flag, and then change the "Served value" option to "a string variant". You will then be prompted to enter a few keys with optional descriptions and set the distribution percentages for each.
+
+Note that the rollout percentage of feature flag variants must add up to 100%. If you wish to exclude some users from your A/B test, configure the **release condition groups**. While the release condition groups determine how many users will be bucketed into **any** of the given variants, the rollout percentage of each variant determines the portion of the overall release group that will be assigned to that particular variant.
+
+### Using A/B feature flags in your code
+
+Getting the value of an A/B flag for the current user is quite simple. With the latest version of our JS library, you can call:
```

Comment:
nit: I think we should avoid proclamations of simplicity and focus on showing how things are achieved. The reader will decide whether we've achieved simplicity :)